include/quda.h

```diff
@@ -685,7 +713,7 @@ extern "C" {
    * @param param  Contains all metadata regarding host and device
    *               storage and solver parameters
    */
-  void incrementalEigQuda(void *_h_x, void *_h_b, QudaInvertParam *param, void *_h_u, double *inv_eigenvals);
+  //void incrementalEigQuda(void *_h_x, void *_h_b, QudaInvertParam *param, void *_h_u, double *inv_eigenvals);
```

Comment:
Should this just be deleted rather than commented out?  It's not good to have things commented out in quda.h: this is the most important file in the library and it needs be clean and definitive 😄